network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -45,16 +46,6 @@
 ]
 
 
-def get_language_code_from_request(request):
```

Comment:
I was going to move this one into utils, only to (re)discover we already had a far better implementation for the same thing there.